extensions/amp-subscriptions/0.1/score-factors.js

```diff
@@ -28,5 +30,6 @@ export const SubscriptionsScoreFactor = {
  * on page config if needed.
  */
 export const DEFAULT_SCORE_CONFIG = {
+  isReadyToPay: 0,
```

Comment:
I thought `isReadyToPay` was a very strong signal to lean towards a platform, shouldn't this be > 10

Reply:
Good question

Reply:
It's one that the publisher gets to decide the weight on - It is a strong signal but how strong is a business decision (as opposed to supportsViewer which is a technology choice). I could make it no default (defaults are optional and missing == 0) to avoid sending the wrong message. I'm open to setting a default, my only concern about that is as we get more signal the interaction gets complex and I really want publishers to decide rather than taking defaults @dvoytenko what do you think?

Reply:
On reflection it shouldn't have a default.  The only reason `supportsViewer` has one is because the login code requires it.  Otherwise the design is that any score factor not specifed by a publisher is ignored so even if a default is set it won't be used.